src/components/ActivityCard.js

```diff
@@ -44,7 +39,6 @@ export default function Card({
 }
 
 const WrapperCard = styled.section`
-  color: rgba(71, 39, 35, 0.72);
   border-bottom: 0.5px solid rgba(71, 39, 35, 0.72);
```

Comment:
pretty thin border...maybe at least 1px! I´ve just remember that gregor said somthing that 0.5px should be avoided!? but not sure

Reply:
damn, completely oversaw this one 😅
Thought I adjusted all of Gregors comments - THANK YOU!